packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -348,15 +357,16 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
         return <div className='trace-context-container'
             onContextMenu={event => this.onContextMenu(event)}
             onKeyDown={event => this.onKeyDown(event)}
+            onKeyUp={event => this.onKeyUp(event)}
```

Comment:
Found an issue with Events Table:

- select a row near the top of the table
- use the scroll bar to navigate to the bottom of the table
- select a row near the bottom of the table
- press Ctrl+Z to undo

This causes a loading of the rows. After loading, the correct row is selected. But Ctrl+Shift+Z to redo does not work. It appears it is a keyboard focus issue, the onKeyDown() isn't being called. If the Events Table title bar is clicked, it appears to give back keyboard focus and then redo works.

If this is found to be an issue not related to this patch, it can be fixed in a separate commit/PR.